tests/integration/components/dwd-rainradar-test.js

```diff
@@ -0,0 +1,27 @@
+import { render } from '@ember/test-helpers';
+import { setupRenderingTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | dwd-rainradar', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`<DwdRainradar />`);
+
+    assert.equal(this.element.textContent.trim(), '');
+
+    // Template block usage:
+    await render(hbs`
+      <DwdRainradar>
+        template block text
+      </DwdRainradar>
+    `);
+
+    assert.equal(this.element.textContent.trim(), 'template block text');
+  });
+});
```

Comment:
we should either delete this boilerplate test file, or write proper tests for the component 😉

Reply:
File removed for now.